firmware/ubirch/rf-sub-GHz/rf_lpuart.h

```diff
@@ -0,0 +1,109 @@
+/*!
```

Comment:
Check the other files and please add C++ wrapper `extern C {...` together with the defines.